tests/spec/w3c/headers-spec.js

```diff
@@ -180,6 +181,33 @@ describe("W3C — Headers", function() {
       expect(doc.querySelector("a[href='http://not-valid']")).toEqual(null);
       expect(doc.querySelector("a[href='http://empty-name']")).toEqual(null);
     });
+
+    it("treats editor's name as HTML", async () => {
+      const config = {
+        specStatus: "REC",
+        editors: [
+          {
+            name:
+              "<span lang='ja'>阿南 康宏</span> (Yasuhiro Anan), (<span lang='ja'>第１版</span> 1st edition)",
+            company: "Microsoft",
+          },
+        ],
+      };
+      const ops = makeStandardOps(config);
+      const doc = await makeRSDoc(ops);
+      const dtElems = [...doc.querySelectorAll(".head dt")];
+      const dtElem = dtElems.find(findEditor);
+      const ddElem = dtElem.nextElementSibling;
+      const [personName, edition] = ddElem.querySelectorAll(
+        "span>span:nth-child(1), span>span:nth-child(2)"
```

Comment:
IMO `span>span` should be enough, but not a blocker.

Reply:
Good point. Will change it.